transportable-udfs-trino-plugin/src/main/java/com/linkedin/transport/trino/TransportUDFClassLoader.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2019 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.transport.trino;
+
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.List;
+
+
+public class TransportUDFClassLoader extends URLClassLoader {
+  private final ClassLoader parent;
+
+  public TransportUDFClassLoader(ClassLoader parent, List<URL> urls) {
+    super(urls.toArray(new URL[0]));
```

Comment:
the modern way to do this is:
```suggestion
    super(urls.toArray(URL[]::new));
```

Reply:
As Transport is still using Java 8, this modern approach seems not allowed.